cmd/commands/runtime_install.go

```diff
@@ -1145,7 +1146,11 @@ func preInstallationChecks(ctx context.Context, opts *RuntimeInstallOptions) (*r
 		return nil, err
 	}
 
-	rt := opts.DownloadRuntimeDef
+	runtimeDef := getRuntimeDef(opts.runtimeDef, opts.versionStr)
+	rt, err := runtime.Download(runtimeDef, opts.RuntimeName, opts.featuresToInstall)
+	if err != nil {
+		return nil, fmt.Errorf("failed to download runtime definition: %w", err)
+	}
```

Comment:
it doesn't make sense to keep the downloaded runtime definition on the opts? i think we can use it later. but if it is not actually being used, i guess there is no point in it

Reply:
I added this in the last version, where I moved runtime definition download to the pre command handler (where we can early check if CLI is compatible with runtime version), but now I noticed that we later check for features such as ingressless...